axum-extra/src/routing/typed.rs

```diff
@@ -80,11 +80,13 @@ use super::sealed::Sealed;
 /// The macro expands to:
 ///
 /// - A `TypedPath` implementation.
-/// - A [`Display`] implementation that interpolates the captures. This can be used to, among other
-/// things, create links to known paths and have them verified statically.
 /// - A [`FromRequest`] implementation compatible with [`RouterExt::typed_get`],
 /// [`RouterExt::typed_post`], etc. This implementation uses [`Path`] and thus your struct must
 /// also implement [`serde::Deserialize`], unless it's a unit struct.
+/// - A [`Display`] implementation that interpolates the captures. This can be used to, among other
+/// things, create links to known paths and have them verified statically. Note that the
+/// [`Display`] implementation for each field must return something that's compatible with it's
```

Comment:
```suggestion
/// [`Display`] implementation for each field must return something that's compatible with its
```

Reply:
haha omg one day I really should learn english 😂